settings/Developer.setting.php

```diff
@@ -48,10 +48,10 @@
     'quick_form_type' => 'YesNo',
     'default' => '0',
     'add' => '4.3',
-    'title' => ts('Enable Drupal Watchdog Logging'),
+    'title' => ts('Send CiviCRM errors to CMS logs?'),
```

Comment:
Nitpick: I'm not a fan of ending labels with a question mark?

Reply:
Well that snowballed quickly